repository-data/application/src/main/resources/hcm-config/configuration/update/Migration20171208.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Can we add a comment describing what this migration does / is for.

Reply:
I will add it in Java/Groovy. Our YAML cleanup script removes all comments :)